Tasks/UseNodeV1/proxyutil.ts

```diff
@@ -0,0 +1,44 @@
+import * as taskLib from 'azure-pipelines-task-lib/task';
+import * as qs from 'querystring';
+import * as url from 'url';
+
+interface ICurlProxy {
+    variable: string,
+    setting: string
+}
+
+function toCurlProxy(proxyCfg: taskLib.ProxyConfiguration): ICurlProxy {
+    let curlProxy: ICurlProxy;
+    if (proxyCfg) {
+        if (proxyCfg && proxyCfg.proxyUrl) {
+            taskLib.debug(`using proxy ${proxyCfg.proxyUrl}`);
+            const parsedUrl = url.parse(proxyCfg.proxyUrl);
+            const httpEnvVarName: string = parsedUrl.protocol === 'https:'? "https_proxy" : "http_proxy";
```

Comment:
will this work since the agent upper cases (matters on linux)? consider changing this to upper case to reflect what really happens

Reply:
It should work, changed it for clarity as suggested.